.github/labeler.yml

```diff
@@ -6,43 +6,52 @@
 # If a label includes a space or a pattern begins with an asterisk or
 # special character, enclose it in quotes.
 
-Breaking:
+'changes existing API':
```

Comment:
I renamed & consolidated a few labels yesterday, so it seems a good time to update this.